updatesnap/SnapVersionModule/snap_version_module.py

```diff
@@ -62,6 +62,32 @@ def process_snap_version_data(upstreamversion, snap_name, version_schema, has_up
     return f"{upstreamversion}-{packagerelease}"
 
 
+def process_rock_version_data(upstreamversion, prevversion, version_schema, has_update):
```

Comment:
`upstream_version` and `previous_version`

Reply:
Updated